includes/wikia/services/ArticleService.class.php

```diff
@@ -177,19 +152,9 @@ private function getTextSnippetSource( $articleId ) {
 			$service = $this;
 			$text = self::$localCache[$articleId] = WikiaDataAccess::cache(
 				$key,
-				86400 /*24h*/,
-				function() use ( $service ) {
-					$content = '';
-					if ( !$this->wg->DevelEnvironment && !empty( $this->wg->SolrMaster ) ) {
-						$content = $service->getTextFromSolr();
-					}
-
-					if ( $content === '' ) {
-						// back-off is to use mediawiki
-						$content = $service->getUncachedSnippetFromArticle();
-					}
-
-					return $content;
+				86400 * 14 /* 14 days, same as parser cache */,
+				function () use ( $service ) {
```

Comment:
Let's avoid assigning `$this` to `$service` and use `$this` in this callback - `use` will be redundant.